crypto/fipsmodule/ec/ec_test.cc

```diff
@@ -1023,6 +1024,16 @@ TEST(ECTest, ArbitraryCurve) {
   // The key must be valid according to the new group too.
   EXPECT_TRUE(EC_KEY_check_key(key2.get()));
 
+  EVP_PKEY *ec_pkey = EVP_PKEY_new();
```

Comment:
we usually use `bssl::UniquePtr` for pointers in the C++ code. did you try that?

Reply:
The sanitizers and valgrind flagged these as leaked.